### PR TITLE
"Hello Microverse" Project

### PR DESCRIPTION
**Pull request by lucy-sees**
In this first pull request you should see the result of the GitHub actions checks

- I updated the linter.yml files
- I updated the .hintrc file
- I updated the json files
- I updated the stylelint file
- I designed the html file as per instructed 
- made sure that all checks are passing
- README file is updated as per microverse guidelines using the Microverse Template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,191 @@
...
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
```
Comment (usorfaitheloho) [inline README.md:50]: According to the recommended Microverse README template given to students, there is an `Installation` and `Future features` section of the file and the only optional section is the  `FAQ` section . Please kindly add these sections back in your README file with the required contents for them 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
## Hi @lucy-sees   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap:  
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Please use more descriptive branch names for your projects to enable reviewers understand the exact feature that was introduced in your PR 😄 
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md
Comment (wayungi) [inline test.md:1]: - [ ] Please delete this file from your repo since it is gitignored and should not be in the repo 👍🏾 
inst

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @lucy-sees , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Atomic commit messages ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Your feature branch name is not descriptive. Please checkout to a new branch and use the correct name and mention the old PR in the new PR. A good branch name describes what features the branch has. ie **hello_microverse-setup**
branch


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**